java/repositories.bzl

```diff
@@ -12,6 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# WARNING: This file only exists for backwards-compatibility.
+# rules_java uses the Bazel federation, so please add any new dependencies to
```

Comment:
Are the repositories in this file already defined in the federation? rules_java_deps() or rules_java_setup()?

Reply:
Yes:
https://github.com/bazelbuild/bazel-federation/blob/master/repositories.bzl#L169
https://github.com/bazelbuild/bazel-federation/blob/master/setup/rules_java.bzl#L3

Reply:
The repositories are here: https://github.com/bazelbuild/bazel-federation/blob/master/java_repositories.bzl

Reply:
Thanks!